packages/devtools_app/lib/src/inspector/inspector_tree_flutter.dart

```diff
@@ -27,13 +29,17 @@ class _InspectorTreeRowWidget extends StatefulWidget {
     @required Key key,
     @required this.row,
     @required this.inspectorTreeState,
+    this.errorText,
   }) : super(key: key);
 
   final _InspectorTreeState inspectorTreeState;
 
   InspectorTreeNode get node => row.node;
   final InspectorTreeRow row;
 
+  /// A description of any error for this row. null if there is no error.
```

Comment:
nit: dartdoc - new line after first sentence

```
/// A description of any error for this row.
///
/// This will be null if there is no error.
```

Reply:
Done, though changed this back to `error` and made it the whole `DevToolsError` so that it can be rendered differently based on whether it's `read`.